wrapper/src/main/java/software/amazon/jdbc/plugin/efm2/MonitorImpl.java

```diff
@@ -326,7 +326,7 @@ public void run() {
     }
 
     LOGGER.finest(() -> Messages.get(
-        "MonitorImpl.startMonitoringThread",
+        "MonitorImpl.stopMonitoringThread",
```

Comment:
is this message name correct? On main I see it's called `MonitorImpl.stopped`

Reply:
I think it's a copy-paste kind of error. It should be a "thread stopped" message. There're two MonitorImpl class. One for efm plugin, the other is for efm2 plugin. You must be checked the class from efm package (that's correct).

https://github.com/aws/aws-advanced-jdbc-wrapper/blob/e90e1aba7e34bb1068789469cd508c35a5cd7b17/wrapper/src/main/java/software/amazon/jdbc/plugin/efm2/MonitorImpl.java#L328